__tests__/app.test.js

```diff
@@ -55,6 +54,7 @@ describe('/api/articles/:article_id', () => {
 						body: 'some gifs',
 						created_at: '2020-11-03T09:12:00.000Z',
 						votes: 0,
+						comment_count: 2,
 					});
 					expect(typeof response.body).toBe('object');
```

Comment:
If you're requesting and checking for a specific article, as you are on line 49, the additional expect statements here are not quite necessary.